instana/instrumentation/sanic_inst.py

```diff
@@ -17,17 +17,21 @@
 
     @wrapt.patch_function_wrapper('sanic.exceptions', 'SanicException.__init__')
     def exception_with_instana(wrapped, instance, args, kwargs):
-        message = kwargs.get("message", args[0])
-        status_code = kwargs.get("status_code")
-        span = async_tracer.active_span
+        try:
+            message = kwargs.get("message") or args[0]
```

Comment:
Would it be more idiomatic to do
```suggestion
            message = kwargs.get("message", args[0])
```
?

Reply:
No, this it the actual bug. This is what I had originally and the issue is that `.get` is a method and it need to have all its arguments evaluated before execution (unfortunately no lazy evaluation ), therefore if you provide message as a keyword argument is going to fire an exception, since it will still attempt to evaluate `arg[0]` which does not exist and it will cause an IndexError

Reply:
Fair enough!